src/Cell.tsx

```diff
@@ -73,7 +74,7 @@ export class Cell<Data: Types.CellBase, Value> extends PureComponent<
     }
   };
 
-  handleMouseOver = (e: SyntheticMouseEvent<*>) => {
+  handleMouseOver = (e: SyntheticEvent<any>) => {
```

Comment:
```suggestion
  handleMouseOver = (e: SyntheticMouseEvent<any>) => {
```
Unclear why this was renamed

Reply:
`SyntheticMouseEvent ` Does not exist.